Billing.API/Services/Invoice/DummyInvoiceProviderService.cs

```diff
@@ -1,16 +1,35 @@
+using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using Billing.API.Models;
 
 namespace Billing.API.Services.Invoice
 {
     public class DummyInvoiceProviderService : IInvoiceService
     {
-        public async Task<IEnumerable<string>> GetInvoices(string clientId) =>
-            await Task.Run(() => new List<string>
+        public async Task<IEnumerable<InvoiceListItem>> GetInvoices(string clientId)
+        {
+            if (clientId.In("000000000000000", "999999999999999"))
+                return await Task.Run(() => new List<InvoiceListItem>());
```

Comment:
Para evitar carga innesesaria:

```suggestion
            if (clientId.In("000000000000000", "999999999999999"))
                return await Task.FromValue(new List<InvoiceListItem>());
```